internal/services/containerapps/helpers/container_apps.go

```diff
@@ -484,7 +484,7 @@ func ContainerAppIngressIpSecurityRestriction() *pluginsdk.Schema {
 				"ip_address_range": {
 					Type:         pluginsdk.TypeString,
 					Required:     true,
-					ValidateFunc: validation.IsCIDR,
+					ValidateFunc: validation.IsIPAddressOrCIDR,
```

Comment:
This can be achieved by specifying multiple validation functions 
```suggestion
					ValidateFunc: validation.Any(validation.IsCIDR, validation.IsIPAddress),
```